packages/client/src/sync/fetcher/storagefetcher.ts

```diff
@@ -27,7 +27,7 @@ import type { Job, SnapFetcherDoneFlags } from './types'
 import type { Debugger } from 'debug'
 const { debug: createDebugLogger } = debugDefault
 
-const TOTAL_RANGE_END = BIGINT_2 ** BIGINT_256 - BIGINT_1
+const TOTAL_RANGE_END = BigInt(BIGINT_2 ** BIGINT_256 - BIGINT_1)
```

Comment:
wouldn't this be bigint anyway?

Reply:
Yes, I think this was something I did while debugging and forgot to clean up. Fixed.